docs/guides/webpack.md

```diff
@@ -15,6 +15,7 @@ labeled as "external", which means they will be ignored:
 
```

Comment:
we should probably just say "the following files" to avoid having to update the number every time.